config.json

```diff
@@ -516,6 +516,17 @@
       "unlocked_by": "robot-name",
       "uuid": "10f091b1-deda-4fec-8e3c-9dabb080d473"
     },
+    {
+      "uuid": "478d8d31-454d-4a8a-af01-96864186d5f2",
+      "slug": "scale-generator",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
+        "strings",
+        "pattern_matching"
```

Comment:
I think the convention is to keep the topics list in alphabetical order... pattern_matching before strings.